pkg/deploymanager/olm.go

```diff
@@ -129,16 +146,6 @@ func (d *DeployManager) CreateOlmResources(olmResources *OlmResources) error {
 		if err != nil && !errors.IsAlreadyExists(err) {
 			return err
 		}
-		csv := &operatorsv1alpha1.ClusterServiceVersion{
-			ObjectMeta: metav1.ObjectMeta{
-				Name:      subscription.Spec.StartingCSV,
-				Namespace: subscription.Namespace,
-			},
-		}
-		err = d.WaitForCsv(csv)
-		if err != nil {
-			return err
-		}
```

Comment:
Have removed this from here as after the operator is installed, In the end, the CheckAllCsvs function will check for all the csvs have succeeded or not. So this is no longer necessary.